src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -110,40 +120,50 @@ const AppsList = (): React.ReactElement => {
-                </StyledLink>
-                {isCustomApp(a.url, appList) && (
-                  <IconBtn
-                    title="Remove"
-                    onClick={(e) => {
-                      e.stopPropagation()
-
-                      setAppToRemove(a)
-                    }}
-                  >
-                    <Icon size="sm" type="delete" color="error" />
-                  </IconBtn>
-                )}
-              </AppContainer>
-            ))}
-        </CardsWrapper>
-
-        <IconText
-          color="secondary"
-          iconSize="sm"
-          iconType="info"
-          text="These are third-party apps, which means they are not owned, controlled, maintained or audited by Gnosis. Interacting with the apps is at your own risk. Any communication within the Apps is for informational purposes only and must not be construed as investment advice to engage in any transaction."
-          textSize="sm"
-        />
+        <SearchInputCard value={appSearch} onValueChange={(value) => setAppSearch(value.replace(/\s{2,}/g, ' '))} />
+        {noAppsFound && <NoAppsFound query={appSearch} />}
+
+        <AnimatePresence>
+          <CardsWrapper>
+            {!appSearch && (
+              <AppCard iconUrl={AddAppIcon} onClick={openAddAppModal} buttonText="Add custom app" iconSize="lg" />
+            )}
+            {apps
+              .filter((a) => a.fetchStatus !== SAFE_APP_FETCH_STATUS.ERROR)
+              .map((a) => (
+                <AppContainer key={a.id} layout initial={{ opacity: 0 }} animate={{ opacity: 1 }} exit={{ opacity: 0 }}>
+                  <StyledLink to={`${appsPath}?appUrl=${encodeURI(a.url)}`}>
+                    <AppCard
+                      isLoading={isAppLoading(a)}
+                      iconUrl={a.iconUrl}
+                      name={a.name}
+                      description={a.description}
+                    />
+                  </StyledLink>
+                  {isCustomApp(a.url, appList) && (
+                    <IconBtn
+                      title="Remove"
+                      onClick={(e) => {
+                        e.stopPropagation()
+
+                        setAppToRemove(a)
+                      }}
+                    >
+                      <Icon size="sm" type="delete" color="error" />
+                    </IconBtn>
+                  )}
+                </AppContainer>
+              ))}
+          </CardsWrapper>
+        </AnimatePresence>
       </ContentWrapper>
+      <CenterIconText
+        color="secondary"
+        iconSize="sm"
+        iconType="info"
+        text="These are third-party apps, which means they are not owned, controlled, maintained or audited by Gnosis. Interacting with the apps is at your own risk. Any communication within the Apps is for informational purposes only and must not be construed as investment advice to engage in any transaction."
```

Comment:
I'd be coherent and use either "app" or "App" here.

Reply:
Sorry, I'm not getting it. The annotation is displayed under the list of apps and applies to all of them. So why would we want it to change from plural to single, or I miss something?

Reply:
I've resolved this after speaking with @mikheevm because it was written by the legal team.